filters/auth/tokenintrospection.go

```diff
@@ -327,6 +405,14 @@ func (f *tokenintrospectFilter) Request(ctx filters.FilterContext) {
 		allowed = f.validateAllClaims(info)
 	case checkOAuthTokenintrospectionAllKV:
 		allowed = f.validateAllKV(info)
+	case checkSecureOAuthTokenintrospectionAnyClaims:
+		allowed = f.validateAnyClaims(info)
+	case checkSecureOAuthTokenintrospectionAnyKV:
+		allowed = f.validateAnyKV(info)
+	case checkSecureOAuthTokenintrospectionAllClaims:
+		allowed = f.validateAllClaims(info)
+	case checkSecureOAuthTokenintrospectionAllKV:
+		allowed = f.validateAllKV(info)
```

Comment:
please group the switch case block: `case checkOAuthTokenintrospectionAnyClaims, checkSecureOAuthTokenintrospectionAnyClaims: ..`

https://play.golang.org/p/Oh7ot21eWrH